docs/config-janitor.rst

```diff
@@ -101,6 +101,10 @@ Additional plugin-specific configuration is needed:
 
     `Optional`: List of unique, user-assigned project IDs (``projectId``) that will be ignored when fetching projects.
 
+.. option:: project_whitelist=["STR","STR"]
+
+    `Optional`: List of unique, user-assigned project IDs (``projectId``) that will be used to fetch instances. If set, janitor will only look at these projects, it will not fetch active projects and the project_blacklist will be ignored.
```

Comment:
I'm not familiar enough with the rst format to know if there's a way to refer back to another parameter for `project_blacklist`, but if so that might be nice. Otherwise maybe quotes or backticks or some other emphasis should be around it to show it's also a config parameter?